repo-projects/docker-creds-provider-8.0.2xx.proj

```diff
@@ -0,0 +1,7 @@
+<Project>
+  <Import Project="docker-creds-provider.proj" />
```

Comment:
Does it make sense to create a common shared project and then two versioned ones vs importing one version from another?  This will make removal/upgrade of 2.2.0 easier.